File: backend/src/test/java/modelo/NotaTest.java
package modelo;

import Elementos.Ing.AgileTasks.modelo.Nota;
import Elementos.Ing.AgileTasks.modelo.Usuario;
import org.junit.Assert;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class NotaTest {
    Nota nota = new Nota();
    Usuario usuario = new Usuario();

    @Test
    public void notaTest(){
        nota.setDescripcion("Descripcion");
        nota.setTitulo("Titulo");
        nota.setUser(usuario);

        assertEquals(nota.getDescrpicion(), "Descripcion");
        assertEquals(nota.getTitulo(), "Titulo");
    }
}

File: backend/src/test/java/modelo/NotaTest.kt
package modelo

import Elementos.Ing.AgileTasks.modelo.Nota
import Elementos.Ing.AgileTasks.modelo.Usuario
import org.junit.Assert
import org.junit.jupiter.api.Test

class NotaTest {

    var nota : Nota = Nota()
    var usuario : Usuario = Usuario()

    @Test
    fun notaTest(){
        nota.setDescripcion("Descripcion")
        nota.titulo = "Titulo"
        nota.setUser(usuario)

        Assert.assertEquals(nota.descrpicion, "Descripcion")
        Assert.assertEquals(nota.titulo, "Titulo")
    }
}